### DISCUSSION TITLE
Error Salsa20

### DISCUSSION THREAD

**Discussion by TeixeiraMauro92** [snipem/gt7dashboard#9] [General]
Building wheels for collected packages: salsa20
  Building wheel for salsa20 (pyproject.toml) ... error
  error: subprocess-exited-with-error

  × Building wheel for salsa20 (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [1 lines of output]
      error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for salsa20
Failed to build salsa20
ERROR: Could not build wheels for salsa20, which is required to install pyproject.toml-based projects

**Comment by snipem** [comment]
I will put this into the README of a future version:

> Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/

I am also thinking about replacing the salsa20 lib with a pure python salsa implementation. This should get rid of the C++ Build Tools requirement on Windows.

**Reply by snipem** [reply]
`salsa20` is now replaced.
